### DISCUSSION TITLE
Missing Canvas option when importing roster into Github Classroom

### DISCUSSION THREAD

**Discussion by dhanishmeht** [community/community#28360] [GitHub Education]
While trying to import the roster into Github Classroom, the only option showing up is Google Classroom, but the documentation mentions Canvas to be an option as well.

**Comment by dockettj** [comment]
Hi Dhanishmeht,

I am also having the same issue. Does your school have a Google Classroom set up that many or most of the teachers use? Some of us use Canvas and others Google Classroom. I am wondering if that is what is limiting us.

**Reply by dhanishmeht** [reply]
My university has Canvas. No google classroom.

**Reply by dockettj** [reply]
Well, it wouldn't be that then. Maybe they took it offline while they update to 1.3?

It's definitely giving me some heartburn. I was using Mimir, but they are shutting down on September 7th and my students are starting back to school on August 22nd. I have a little bit of runway, but was hoping to have everything rebuilt on day 1.

**Comment by kanazuchijin** [comment]
> While trying to import the roster into Github Classroom, the only option showing up is Google Classroom, but the documentation mentions Canvas to be an option as well.

I am experiencing the same issue. Canvas is no longer listed as an option; Moodle and Sakai used to be options as well. It appears to be an issue with the version of Learning Tools Interoperability (LTI). When I try to import a roster from my LMS, I see this message displayed next to the option to manually upload a roster:

> We are updating our Learning Tools Interoperability (LTI) to v1.3. The existing LMS Canvas, Moodle, Sakai or other LMS intergration will ask to upload the roster manually before that.

If I read this correctly, it appears that GitHub Classroom is using an older version of LTI. This means those of us not using Google Classroom will have to upload the roster manually until LTI is updated to 1.3.

**Comment by pinventado** [comment]
Same issue with connecting to Canvas.

This is frustrating because people are not informed before performing such changes. Similar cases have happened in the past when there were big changes made to the system. I would have thought communication to users improved by now. This really needs to change.

**Comment by dockettj** [comment]
Hey everyone, I found a clear answer.

https://github.com/community/Global-Campus-Teachers/discussions/104

Hope that helps!

**Reply by pinventado** [reply]
Can't seem to open the link. Is that a private group?

**Reply by dockettj** [reply]
Clicking the link takes me to a 404, too. However, it is the right link. Hover over it and then click the title "LMS removed from Roster Import..."

**Reply by dfridovi** [reply]
Confirming that I am only seeing the google classroom button, and getting 404 on the above link. Would be super nice to get this resolved before class starts next week...

**Reply by dockettj** [reply]
@dfridovi hover over the link and then click "LMS removed from roster import." Still only seeing Google, too. I have a support ticket in and they said Canvas moved away from LTI 1.0 this summer which broke support. They are updating, but do not have an ETA.

**Reply by pinventado** [reply]
Hmm, I don't quite get what you mean by hovering over. If I leave my mouse on top of the link I don't see a popup or anything (if that's what you are seeing). Do you have a link to the new page it links to?

**Reply by domoritz** [reply]
I think this is the link: `https://github.com/community/Global-Campus-Teachers/discussions/104`

**Reply by pinventado** [reply]
Still a 404 for me ... maybe I'm not a member of that community. I thought I was.

**Reply by ghost** [reply]
404 for me too.  @dockettj , maybe you could give us a TL;DR?

**Reply by domoritz** [reply]
They are discussing the issue with 1.3 but no response on when we may get a fix for the last two weeks.
